compatibility/compatibility-vec-common/src/main/java/com/foursoft/harness/compatibility/vec/common/VecCreator.java

```diff
@@ -48,7 +48,8 @@ public final class VecCreator {
 
     private static final Map<String, VecVersion> CLASS_PATH_TO_VEC_VERSION_MAP = Map.ofEntries(
             Map.entry("com.foursoft.harness.vec.v113.VecContent", VecVersion.VEC11X),
-            Map.entry("com.foursoft.harness.vec.v12x.VecContent", VecVersion.VEC12X)
+            Map.entry("com.foursoft.harness.vec.v12x.VecContent", VecVersion.VEC12X),
+            Map.entry("com.foursoft.harness.vec.v2x.VecContent", VecVersion.VEC20X)
```

Comment:
This MR seems more like refactoring and contains nothing about VEC 2.0 so I wouldn't expect those changes.